convert/convert/config.py

```diff
@@ -29,22 +29,22 @@
 class Settings(BaseSettings):
     """Base settings values"""
 
-    minio_host: Optional[str] = os.getenv("MINIO_HOST")
-    minio_access_key: Optional[str] = os.getenv("MINIO_ACCESS_KEY")
-    minio_secret_key: Optional[str] = os.getenv("MINIO_SECRET_KEY")
+    minio_host: Optional[str] = os.getenv("S3_ENDPOINT_URL")
+    minio_access_key: Optional[str] = os.getenv("S3_ACCESS_KEY")
+    minio_secret_key: Optional[str] = os.getenv("S3_SECRET_KEY")
     s3_prefix: Optional[str] = os.getenv("S3_PREFIX")
-    s3_credentials_provider: Optional[str] = os.getenv(
-        "S3_CREDENTIALS_PROVIDER", "minio"
-    )
+    s3_credentials_provider: Optional[str] = os.getenv("S3_PROVIDER", "minio")
```

Comment:
Could we rename these according to the new naming of env vars?

Reply:
Renamed